test/ctia/bundle/core_asset_ref_entities_test.clj

```diff
@@ -106,44 +87,61 @@
            (is (every? (partial contains? asset-refs) refs))))))))
 
 (deftest validate-asset-refs-test
-  (testing "Bundle with asset_refs that have no correspoding Asset"
+  (testing "Bundle with asset_refs that have no corresponding Asset"
     (let [;; assign non-transient ID to the asset in the Bundle,
-          ;; leaving :asset_refs pointing to a transient ID that would never resolve
-          bundle (walk/prewalk
-                  #(if (and (map? %)
-                            (-> % :type (= "asset")))
-                     (assoc % :id "http://ex.tld/ctia/asset/asset-61884b14-e273-4930-a5ff-dcce69207724")
-                     %)
-                  bundle-ents)]
-      (th/fixture-ctia-with-app
+          ;; leaving :asset_ref's pointing to a transient ID that would never resolve
+          bundle (update bundle-ents :assets
+                         #(into #{} (map (fn [asset]
+                                           (assoc asset :id "http://ex.tld/ctia/asset/asset-61884b14-e273-4930-a5ff-dcce69207724")))
+                                %))]
+      (test-for-each-store-with-app
        (fn [app]
-         (let [services (app/service-graph app)]
-           (is
-            (thrown? Exception
-             (bundle/import-bundle
-              bundle
-              nil         ;; external-key-prefixes
-              login
-              services))))))))
+         (th/set-capabilities! app "foouser" ["foogroup"] "user" (all-capabilities))
+         (whoami-helpers/set-whoami-response app
+                                             "45c1f5e3f05d0"
+                                             "foouser"
+                                             "foogroup"
+                                             "user")
+         (let [create-response (th/POST app
```

Comment:
🟠  why did you need to use the http layer (+ authorization) instead of the existing `bundle/import`?

Reply:
I couldn't figure out how to make the test run on my machine.